logs/analyze.go

```diff
@@ -1680,35 +1689,68 @@ func classifyAndSetDetails(logLine state.LogLine, statementLine state.LogLine, d
 				"vacuum_page_dirty": vacuumPageDirty, "read_rate_mb": readRateMb,
 				"write_rate_mb": writeRateMb, "rusage_kernel": rusageKernelMode,
 				"rusage_user": rusageUserMode, "elapsed_secs": rusageElapsed,
-				"pinskipped_pages": pinskippedPages, "frozenskipped_pages": frozenskippedPages,
 			}
 			// List anti-wraparound status either if the message indicates that it is, or if
 			// our Postgres version is new enough (13+) as determined by the presence of WAL
-			// record information (parts[26])
+			// record information (parts[37])
```

Comment:
Note for reviewer: In general most part numbers after the newly added lines have incremented by 10 - this here got incremented by 11 because there was a miss on a previous change that affected part numbers but forgot to adjust the comment.